### DISCUSSION TITLE
[UNSTABLE] Submarines belonging to BOTH teams spawn in outpost deathmatch

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#14282] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Outpost deathmatch spawns 2 submarines, one docked. The docked sub spawns like a completely normal submarine... which is problematic because it has a fully autofilled medbay and armory, meaning the team that reaches it first generally wins.

### Reproduction steps

1. Start a game of outpost deathmatch
2. Observe.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.101.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and fix tested to be working correctly. Fix coming in the next Unstable build. Closing.
